Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -45,11 +45,13 @@ private string GetInfoText()
                 return string.Empty;
             }
 
+            var playerManager = IoCManager.Resolve<IPlayerManager>(); ///Can this be in playerCount?
```

Comment:
Should use a dependency. check if it's already in scope as _playerManager

Reply:
Thanks, it was in the scope